docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+# Fine-tune Models
+
+Although CLIP-as-service has provided you a list of pre-trained models, you can also train your models. 
+This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fine-tune models and use them in CLIP-as-service.
+
+For installation and basic usage of Finetuner, please refer to [Finetuner website](https://finetuner.jina.ai).
+
+## Prepare Training Data
+
+Finetuner accepts training data and evaluation data in the form of [`DocumentArray`](https://docarray.jina.ai/fundamentals/documentarray/).
+The training data for CLIP is a list of (text, image) pairs.
```

Comment:
Can we add a data sample here? I feel it is not easy to understand what's the dataset should looks like. I belive a real data sample could help the reader.

Reply:
let me think of an elegant way